### PR TITLE
Adoc Updates

### PR DESCRIPTION
**Pull request by joefol**
Steps for creating a Micronaut application and configurations for using Pi4Micronaut.

### CODE REVIEW

**Review by SwindleA — APPROVED** [review]
looks good

File: pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+==== Add Dependency
+. Go to the build.gradle file and find the dependencies.
+. Add the Pi4Micronaut dependency to the list.
++
+[source, gradle]
+----
+implementation("io.github.oss-slu:pi4micronaut-utils:1.0:all")
+----
++
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc:36]: There is an additional + sign here.

File: pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+It should look like:
+image:Adding_Dependency.png[]
+
+==== Setup Configuration
+. Go to the configuration file:
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc:41]: Make this more clear. Go to the application.yml file to add configuration of  your circuit setup

File: pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    digital-output:
+        led:
+            name: LED
+            address: 17
+            shutdown: LOW
+            initial: LOW
+            provider: pigpio-digital-output
+----
++
+Here we are specifying a led as a digital output type with name LED, address 17, etc.
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc:58]: add a link to led documentation here for more info about configuration

File: pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
++
+It should look like:
+image:Add_Configuration.png[]
+
+==== Creating Component Class
+. Create a new class in the com.demoPi4Micronaut directory
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc:64]: Create a new class in the project's src directory as shown below.

File: pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
++
+path: Demo_Pi4Micronaut/src/main/java/com.demoPi4Micronaut
++
+This is where code will be written to implement a LED light using the Pi4Micronaut library.
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc:68]: This is where you use the implemented LED functions from Pi4Micronaut library.

File: pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+. Our example code is:
++
+image:Add_Class.png[]
+
+. Now that the code for the LED has been written, see our other documentation on building and running
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc:74]: Since the two sections are next to each other, we don't need this

**Review by ruthvikm — CHANGES_REQUESTED** [review]

File: pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+==== Creating Component Class
+. Create a new class in the project's src directory as shown below.
++
+path: Demo_Pi4Micronaut/src/main/java/com.demoPi4Micronaut
++
+This is where you use the implemented LED functions from Pi4Micronaut library.
+
+. Our example code is:
++
```
Comment (SwindleA) [inline pi4micronaut-utils/src/docs/asciidoc/Introduction/howToUsePi4Micronaut.adoc:71]: It would be great if the user could copy paste the example code.

**Review by SwindleA — APPROVED** [review]

**Review by ruthvikm — CHANGES_REQUESTED** [review]

**Review by ruthvikm — APPROVED** [review]
Looks good.

**Pull request merged**